packages/hdwallet-ledger-webusb/src/transport.ts

```diff
@@ -13,15 +12,24 @@ import {
   LedgerTransportMethod,
   LedgerTransportMethodName,
 } from "hdwallet-ledger/src/transport";
+import PQueue from "p-queue/dist";
+
+import { VENDOR_ID } from "./adapter";
 
 const RECORD_CONFORMANCE_MOCKS = false;
 
+const callsQueue = new PQueue({ concurrency: 1, interval: 1000 });
+
 export async function getFirstLedgerDevice(): Promise<USBDevice | null> {
   if (!(window && window.navigator.usb)) throw new core.WebUSBNotAvailable();
 
   const existingDevices = await TransportWebUSB.list();
 
-  return existingDevices.length > 0 ? existingDevices[0] : null;
+  return existingDevices.length > 0
+    ? existingDevices[0]
+    : window.navigator.usb.requestDevice({
+        filters: [{ vendorId: VENDOR_ID }],
+      });
```

Comment:
The magic - this previously assumed a connected device with permission already granted, this will now request it otherwise